pkg/nfd-master/nfd-api-controller.go

```diff
@@ -54,6 +55,8 @@ func newNfdController(config *restclient.Config, nfdApiControllerOptions nfdApiC
 	}
 
 	nfdClient := nfdclientset.NewForConfigOrDie(config)
+	utils.KlogDump(2, "NFD API controller options:", "  ", nfdApiControllerOptions)
```

Comment:
We need to make the fields in `nfdApiControllerOptions` public so that this dump will print something